Dockerfile

```diff
@@ -6,17 +6,12 @@
 FROM golang:1.14.3-alpine AS build
 RUN apk add git bash make
 RUN mkdir /app
+COPY . /app
```

Comment:
Yup, I'm aware of the optimization. Happy to go back to considering `go.mod` and `go.sum` separately for caching purposes. However, in terms of build correctness, I would rather prefer if we re-used the `Makefile` targets instead of maintaining two sources of truth (i.e., the `Makefile` and the `Dockerfile`).

Reply:
The make targets seem fine to me (i.e., `RUN make go-mod-download` rather than `RUN go mod download` and `RUN make interuss` rather than `RUN go install ./...`), but I do think we want to keep the other command orderings separate copying for build optimization.